src/main/java/com/flightstats/hub/dao/aws/S3Verifier.java

```diff
@@ -146,7 +146,7 @@ VerifierRange getSingleVerifierRange(DateTime now, ChannelConfig channelConfig)
     }
 
     @VisibleForTesting
```

Comment:
I still think this should be removed :-) 
As per this javadoc https://google.github.io/guava/releases/19.0/api/docs/com/google/common/annotations/VisibleForTesting.html
this annotation gives wide visibility to the member for testing purpose. Java access modifier is already taking care of the visibility. This is redundant.

Reply:
The annotation doesn't change accessibility of the method, though. It just notes that it is more widely accessible than it needs to be...and it's like that only for testing purposes. It also has a side effect of IntelliJ no longer complaining that you're using the wrong accessibility level.